lib/cobra_commander/dependencies/bundler.rb

```diff
@@ -33,10 +33,12 @@ def lockfile
         @lockfile ||= ::Bundler::LockfileParser.new(::Bundler.read_file(path))
       end
 
-      def components_source
-        @components_source ||= begin
-          source = @lockfile.sources.find { |s| s.path.to_s.eql?("components") }
-          ::Bundler::Source::Path.new(source.options.merge("root_path" => @root))
+      def components_sources
+        @components_sources ||= @lockfile.sources.filter_map do |source|
```

Comment:
Any possibility that the `Gemfile` has two path blocks and the first is not the path for internal components? Or, at least, is it worth documenting this behavior in the README? e.g "Put your internal component path block first"

Reply:
This actually changes the behavior to look at all path sources instead of only the first one in `components`.

The assumption here is that every gem stored locally (in a path source, not in rubygems or any external source) belongs to the umbrella.

Reply:
ahh -- I see what you mean.